src/BookReader/Mode1UpLit.js

```diff
@@ -0,0 +1,442 @@
+// @ts-check
+import { customElement, html, LitElement, property, query } from 'lit-element';
+import { styleMap } from 'lit-html/directives/style-map';
+import { ModeSmoothZoom } from './ModeSmoothZoom';
+import { arrChanged, calcScreenDPI, genToArray, sum, throttle } from './utils';
+import { HTMLDimensionsCacher } from "./utils/HTMLDimensionsCacher";
+/** @typedef {import('./BookModel').BookModel} BookModel */
+/** @typedef {import('./BookModel').PageIndex} PageIndex */
+/** @typedef {import('./BookModel').PageModel} PageModel */
+/** @typedef {import('./ModeSmoothZoom').SmoothZoomable} SmoothZoomable */
+/** @typedef {import('./PageContainer').PageContainer} PageContainer */
+/** @typedef {import('../BookReader').default} BookReader */
+
+// I _have_ to make this globally public, otherwise it won't let me call
+// it's constructor :/
+/** @implements {SmoothZoomable} */
+@customElement('br-mode-1up')
+export class Mode1UpLit extends LitElement {
+  /****************************************/
+  /************** PROPERTIES **************/
+  /****************************************/
+
+  /** @type {BookReader} */
+  br;
+
+  /************** BOOK-RELATED PROPERTIES **************/
+
+  /** @type {BookModel} */
+  @property({ type: Object })
+  book;
+
+  /** @type {PageModel[]} */
+  @property({ type: Array })
+  pages = [];
+
+  /** @type {{ [pageIndex: string]: number }} */
```

Comment:
```suggestion
  /** @type {Record<PageIndex, number>} */
```

Confirm if this is inches

Reply:
Yep. Done.